contrib/utilities/indent-all

```diff
@@ -25,6 +25,8 @@
 #
 # The script needs to be executed as
 #   ./contrib/utilities/indent-all
+# or
+#   ./contrib/utilities/indent-all clang_format_command
```

Comment:
as above

Reply:
I don't understand the line:
export DEAL_II_CLANG_FORMAT="${CLANG_FORMAT_COMMAND:-clang-format}"
How about the committed code?
It checks if the environment variable DEAL_II_CLANG_FORMAT exists. If it does not exists it assign the variable to "clang-format"

I tested and it works:
 export DEAL_II_CLANG_FORMAT=clang-format-6.0
 ./contrib/utilities/indent

On the other hand. This does not work (I'm not familiar with cmake environment variables):
make -DDEAL_II_CLANG_FORMAT=clang-format-6.0